api/meter.go

```diff
@@ -228,6 +258,38 @@ func (r *relayMeter) AppRelays(app string, from, to time.Time) (AppRelaysRespons
 	return resp, nil
 }
 
+func (r *relayMeter) AppLatency(app string) (AppLatencyResponse, error) {
+	r.Logger.WithFields(logger.Fields{"app": app}).Info("apiserver: Received AppLatency request")
+
+	resp := AppLatencyResponse{
+		Application:  app,
+		DailyLatency: r.todaysLatency[app],
+		From:         r.todaysLatency[app][0].Time,
+		To:           r.todaysLatency[app][23].Time,
+	}
+
+	return resp, nil
+}
+
+func (r *relayMeter) AllAppsLatency() ([]AppLatencyResponse, error) {
+	r.Logger.Info("apiserver: Received AllAppsLatency request")
+
+	resp := []AppLatencyResponse{}
+
+	for app, appLatency := range r.todaysLatency {
+		latencyResp := AppLatencyResponse{
+			Application:  app,
+			DailyLatency: appLatency,
+			From:         appLatency[0].Time,
+			To:           appLatency[23].Time,
```

Comment:
Better to use the length of `appLatency` to avoid hard-coding the expected length.
Also are these latency entries guaranteed to be sorted based on Time?
Sorting the entries here is probably needed to avoid making any assumptions on returned data elsewhere.

Reply:
Updated to use latency and ensure values are sorted by date in ascending order.